OpenXLSX/sources/XLCellValue.cpp

```diff
@@ -217,14 +217,18 @@ XLCellValueProxy& XLCellValueProxy::setError()
     // ===== Set the type to "e", i.e. error
     m_cellNode->attribute("t").set_value("e");
 
-    // ===== Remove the value node, if it exists
-    m_cellNode->remove_child("v");
+    // ===== If the cell node doesn't have a value attribute, create it.
+    if (!m_cellNode->attribute("v")) m_cellNode->append_attribute("v");
```

Comment:
Why does the cell node need a value attribute? As far as I know, this is not part of the Excel specification.

Reply:
It's probably not needed, you're right. It didn't hurt though (tests - apart from the test I added in testcases - were manual opening of Excel files, and checking cell values with ISERR or ISNAN).